src/components/flowchart-wrapper/flowchart-wrapper.js

```diff
@@ -358,6 +360,7 @@ export const FlowChartWrapper = ({
 
 export const mapStateToProps = (state) => ({
   fullNodeNames: getNodeFullName(state),
+  displaySidebar: state.display.sidebar,
```

Comment:
Is sidebarVisible always false if displaySidebar is false ?

Reply:
Yes, as displaySidebar removes sidebar completely where sidebarVisible just collapse sidebar. So when  
displaySidebar is false, sidebarVisible is ignored.